test/backend_test.in.cpp

```diff
@@ -5518,6 +5518,38 @@ NGRAPH_TEST(${BACKEND_NAME}, dequantize)
               read_vector<output_c_type>(y));
 }
 
+NGRAPH_TEST(${BACKEND_NAME}, dequantize_cpu)
```

Comment:
why just a test for Dequantize and not Quantize?  Maybe existing tests for Quantize round mode, etc. covered it?